docs/faqs/max_execution_time.rst

```diff
@@ -8,31 +8,58 @@ To ensure fairness, there is a maximum execution time for each Qiskit Runtime jo
 a job exceeds this time limit, it is forcibly cancelled. This is represented in the job
 status as `Cancelled - Ran too long`. The maximum execution time is the
 smaller of 1) the system limit and 2) the ``max_execution_time`` defined by the program.
+
+System limit
+--------------
+
 The system limit is defined as follows:
 
 Qiskit Runtime on IBM Cloud
----------------------------
+***************************
 
 The system limit on the job execution time is 3 hours for a job running on a simulator
 and 8 hours for a job running on a physical system.
 
 Qiskit Runtime on IBM Quantum
------------------------------
+*****************************
 
-The system limit on the job execution time is
+The system limit on the job execution time is described in the following table:
 
 +------------------+--------------+-----------+--------------+-----------+
 |                  | Public Program           | Private Program          |
 +==================+==============+===========+==============+===========+
 |                  | Premium User | Open User | Premium User | Open User |
 +------------------+--------------+-----------+--------------+-----------+
-| Simulated Device | 3h           | 1h        | 3h           |1h         |
+| Simulated Device | 3h           | 1h        | 3h           | N/A       |
 +------------------+--------------+-----------+--------------+-----------+
-| Real Device      | 8h           | 4h        | 8h           |2h         |
+| Real Device      | 8h           | 4h        | 8h           | N/A       |
 +------------------+--------------+-----------+--------------+-----------+
 
 Note that a *premium user* here means a user who has access to backends in providers other than ibm-q/open/main.
 
+Max execution time
+------------------
+
+There are several factors that impact the program's maximum execution time. In addition to the ``max_execution_time`` parameter, there are time limits on programs and sessions.
+
+Sessions
+********
+
+When a session is started, it is assigned a maximum session timeout value.  After the maximum session timeout is reached, the session is permanently closed. The maximum session timeout value is set on the ``max_time`` parameter, which can be greater than the program’s ``max_execution_time``. By default, it is set to the  system limit.
+
+Additionally, there is a 5 minute *interactive* timeout value. If there are no session jobs queued within that window, the session is temporarily deactivated and normal job selection resumes. After the new session becomes inactive, if the job scheduler gets a job from the original session and its maximum timeout value has not been reached, the session is reactivated until its maximum timeout value is reached.
```

Comment:
I think this paragraph is missing something. Where did the "new session" come from?